webboot/webboot.go

```diff
@@ -212,9 +212,12 @@ func main() {
 			log.Fatal(err)
 		}
 	}
-
+	//the Request function sets up a DHCP confifuration for all interfaces,
+	//such as eth0, which is an ethernet interface.
 	if *ipv4 || *ipv6 {
 		dhclient.Request(*ifName, *timeout, *retry, *verbose, *ipv4, *ipv6)
+
+		//*ifname is used as the name of the interface, defaulting with ^[we].*
```

Comment:
if you can, put the comment above the call i.e. before line 218. And, note, ... "name" is not right. It's singular. So now you need to dig a bit and see ... could it be more than one? If so, why?

Reply:
👍